modules/apps/object/object-test/src/testFunctional/paths/ObjectField.path

```diff
@@ -119,7 +119,7 @@
 	</tr>
 	<tr>
 		<td>TYPE_OPTION</td>
-		<td>//button[@class='dropdown-item']//div[contains(text(),"${key_fieldType}")]</td>
+		<td>//button[@class='dropdown-item']//div[(text()="${key_fieldType}")]</td>
 		<td></td>
 	</tr>
```

Comment:
If we're making changes to a path, we should be doing this in a separate commit with a clear commit message explaining why. I'm only expecting a new automated test and new macros/paths for the test in this commit.

@andreefarias please follow this convention from now on, I'll be enforcing this in the future.